src/workerd/tests/BUILD.bazel

```diff
@@ -84,3 +84,34 @@ wd_cc_benchmark(
     srcs = ["bench-fast-api.c++"],
     deps = ["//src/workerd/jsg"],
 )
+
+BENCHMARKS = [
+    ":bench-json",
+    ":bench-mimetype",
+    ":bench-kj-headers",
+    ":bench-api-headers",
+    ":bench-global-scope",
+    ":bench-regex",
+    ":bench-fast-api",
+]
+
+filegroup(
+    name = "all_benchmarks",
+    srcs = BENCHMARKS,
+    visibility = ["//visibility:public"],
+)
+
+genrule(
+    name = "run_all_benchmarks",
+    outs = ["run_benchmarks.sh"],
+    cmd = """
+        echo '#!/bin/bash' > $@
+        echo 'set -e' >> $@
+        echo 'echo "Running all benchmarks..."' >> $@
+        for bench in %s; do
+            echo "bazel run //src/workerd/tests:$${bench#:}" >> $@
```

Comment:
```suggestion
            echo "bazel run //src/workerd/tests:$${bench#:} --config=benchmark" >> $@
```

Reply:
A target like this is not Bazel style at all. Why not just run the tests using `bazel test <args> //src/workerd/tests: all_benchmarks` instead of building a script?

Reply:
benchmarks are a cc_binary not a test. bazel test won't work.